docs/ecs_guide.md

```diff
@@ -259,42 +282,70 @@ if __name__ == "__main__":
 
-## Next steps
-
-Now that you are confident your ECS worker is healthy, you can experiment with different work pool configurations.
+```yaml
+name: ecs-worker-guide
+# this is pre-populated by running prefect init
+prefect-version: 2.14.20 
 
-- Do your flow runs require higher `CPU`?
-- Would an EC2 `Launch Type` speed up your flow run execution?
+# build section allows you to manage and build docker images
+build:
+- prefect_docker.deployments.steps.build_docker_image:
+    id: build_image
+    requires: prefect-docker>=0.3.1
+    image_name: <my-ecr-repo>
+    tag: latest
+    dockerfile: auto
 
-These infrastructure configuration values can be set on your ECS work pool or they can be overridden on the deployment level through [job_variables](https://docs.prefect.io/concepts/infrastructure/#kubernetesjob-overrides-and-customizations) if desired.
+# push section allows you to manage if and how this project is uploaded to remote locations
+push:
+- prefect_docker.deployments.steps.push_docker_image:
+    requires: prefect-docker>=0.3.1
+    image_name: '{{ build_image.image_name }}'
+    tag: '{{ build_image.tag }}'
+
+ # the deployments section allows you to provide configuration for deploying flows
+deployments:
+- name: my_ecs_deployment
+version:
+tags: []
+description:
+entrypoint: flow.py:my_flow
+parameters: {}
+work_pool:
+    name: ecs-dev-pool
+    work_queue_name:
+    job_variables:
+    image: '{{ build_image.image }}'
+schedules: []
+pull:
+- prefect.deployments.steps.set_working_directory:
+    directory: /opt/prefect/ecs-worker-guide
 
-Consider adding a [build action](https://docs.prefect.io/concepts/deployments-ux/#the-build-action) to your [`prefect.yaml`](https://docs.prefect.io/concepts/deployments-ux/#the-prefect-yaml-file) file if you want to automatically build a Docker image and push it to an image registry. Note that a Docker image is built and pushed by default if a deployemnt is creaded with the `flow.deploy` method in Python.
+```
 
-Here is an example build action for ECR:
+### 2. [Deploy](https://docs.prefect.io/tutorial/deployments/#create-a-deployment) the flow to the server, specifying the ECS work pool when prompted
 
-```yaml
-build:
-- prefect.deployments.steps.run_shell_script:
-    id: get-commit-hash
-    script: git rev-parse --short HEAD
-    stream_output: false
-- prefect.deployments.steps.run_shell_script:
-    id: ecr-auth-step
-    script: aws ecr get-login-password --region <region> | docker login --username
-        AWS --password-stdin <>.dkr.ecr.<region>.amazonaws.com
-    stream_output: false
-- prefect_docker.deployments.steps.build_docker_image:
-    requires: prefect-docker>=0.3.0
-    image_name: <your-AWS-account-number>.dkr.ecr.us-east-2.amazonaws.com/<registry>
-    tag: '{{ get-commit-hash.stdout }}'
-    dockerfile: auto
-    push: true
+```bash
+prefect deploy my_flow.py:my_ecs_deployment
 ```
+
+### 3. Find the deployment in the UI and click the **Quick Run** button!
```

Comment:
```suggestion
### 5. Find the deployment in the UI and click the **Quick Run** button!
```
Maybe also add a screenshot here if we're mentioning a specific button in the UI?